src/lib/components/metrics-definition/MetricsDefinitionGenerateButton.svelte

```diff
@@ -25,12 +27,29 @@
     tooltipText = undefined;
     buttonDisabled = false;
   }
+
+  let modalIsOpen = false;
+
+  const openModal = () => {
+    console.log("opening");
```

Comment:
Can we remove this log?

Reply:
oops, yes, thanks for catching these @djbarnwal ! :-)